src/builder/QuotecallWrapper.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity ^0.8.23;
+
+import {CodeJarHelper} from "./CodeJarHelper.sol";
+import {IQuarkWallet} from "quark-core/src/interfaces/IQuarkWallet.sol";
+import {Quotecall} from "../Quotecall.sol";
+
+// Helper library to wrap a QuarkOperation from Actions.sol for a Paycall
+library QuotecallWrapper {
+    function wrap(IQuarkWallet.QuarkOperation memory operation, uint256 chainId, uint256 quotedAmount)
```

Comment:
Same comment as in #12. `chainId` can be removed.

Reply:
I think wrap() probably will still need chainId now as we need that to find the right address for pricefeed and payment token for compute the address for quotecall.